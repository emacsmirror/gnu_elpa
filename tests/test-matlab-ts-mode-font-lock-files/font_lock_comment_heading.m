% -*- matlab-ts -*-
function [a, b] = test_comment_heading
% help for test_comment_heading
%
% % isn't a section heading in the doc comment
%

    % text before heading
    %% comment heading 1
    %
    % some comment
    %
    %% comment heading 2 within one comment
    %
    % some text

    a = 1;

    %% comment heading 3

    b = 1;
end
