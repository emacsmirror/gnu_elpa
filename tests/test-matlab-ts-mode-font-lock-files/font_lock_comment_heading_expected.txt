c ccc ccccccccc ccc
kkkkkkkk bvD vb d ffffffffffffffffffff
h hhhh hhh hhhhhhhhhhhhhhhhhhhh
h
h h hhhhh h hhhhhhh hhhhhhh hh hhh hhh hhhhhhh
h

    c cccc cccccc ccccccc
    HH HHHHHHH HHHHHHH H
    c
    c cccc ccccccc
    c
    HH HHHHHHH HHHHHHH H HHHHHH HHH HHHHHHH
    c
    c cccc cccc

    d d nD

    HH HHHHHHH HHHHHHH H

    d d nD
kkk
